src/Permissions/Backend/App.vala

```diff
@@ -141,21 +141,15 @@ public class Permissions.Backend.App : GLib.Object {
         );
     }
 
-    private string negate_permission (string permission) {
-        var new_permission = permission;
+    private bool is_permission_overridden (GenericArray<string> overrides, string permission) {
+        var negated_permission = permission;
```

Comment:
It is not necessary to initialise `negated_permission` - it must get initialised in the following lines.  So it would be better not use this confusing assignment (something negated cannot equal itself)